### PR TITLE
CSS problem with overlapping text with icon

### PR DESCRIPTION
**Pull request by Culprit-CZ**
http://phpbbsocialnetwork.com/viewtopic.php?f=44&t=4417

### CODE REVIEW

**Comment by Kamahl19** [comment]
MA

File: sn-theme/prosilver/classes_userstatus.less [hunk 1]
```diff
@@ -193,10 +193,12 @@
 			right: 1;
 		}
 
+
```
Comment (senky) [inline sn-theme/prosilver/classes_userstatus.less]: unnecessary line

File: sn-theme/prosilver/classes_userstatus.less [hunk 2]
```diff
@@ -193,10 +193,12 @@
...
     & > div{
       color: #fff;
       white-space: pre-wrap;
       width: 100%;
+
```
Comment (senky) [inline sn-theme/prosilver/classes_userstatus.less]: unnecessary line

**Comment by senky** [comment]
PR lacks padding for ltr.

Plus, we agreed on using `visibility:hidden;` instead of `display: none` so that padding works also for hidden "X".

**Comment by Culprit-CZ** [comment]
But `:not` is not a part of this PR.
Otherwise `:not` is not suporting only IE8

**Comment by senky** [comment]
It is part of this PR:
https://github.com/phpBB-Social-Network/phpBB-Social-Network/pull/134/files#L0R947

What is the fallback for IE8? It is still commonly user browser, if `:not` is not supported there, how that parts of page will be displayed correctly?

**Comment by Culprit-CZ** [comment]
Is it not, It's only your comments, that put it in this PR. Otherweise is it not part of this PR.
Did you tested?
We have tested on all browsers. Also .....

**Comment by senky** [comment]
I just do not understand. I clicked Files changed tab and looked at the code. In root/styles/prosilver/theme/socialnet.css on line 947 I can see **new** code added using `:not` selector. What Is what my comment put into this PR?

**Comment by Culprit-CZ** [comment]
File `root/styles/prosilver/theme/socialnet.css` changes are irelevant, because this file is each time fully compiled from less files. And as you see, in less files is not added `:not`.

**Comment by senky** [comment]
Well, this is not the way I like it, nor I imagined the solution, but it works and fixes the bug, so:

MA

**Comment by Kamahl19** [comment]
MA

**Pull request merged**